pkg/tanka/inline.go

```diff
@@ -69,6 +69,11 @@ func (i *InlineLoader) Load(path string, opts JsonnetOpts) (*v1alpha1.Environmen
 	return env, nil
 }
 
+func (i *InlineLoader) Peek(path string, opts JsonnetOpts) (*v1alpha1.Environment, error) {
+	opts.EvalScript = EnvsOnlyEvalScript
```

Comment:
Can we check if this was already set upstream? It would make this function much more versatile.

```suggestion
	if opts.EvalScript == "" { opts.EvalScript = EnvsOnlyEvalScript }
```

Reply:
I don't think this would actually be beneficial. The interface defines that this function returns environment metadata without resources, so consumers can rely on that

By doing this, it would become possible to make the implementation break the interface assumption based on the values you pass.

Also, if you really need such functionality, you could just use:

```go
tanka.InlineLoader{}.Load(path, jsonnet.Opts{EvalScript: "myScript"})
```

Reply:
`Peek` makes the promise of returning an `Environment` object, as that object can have `data`, I'd argue that can be populated depending on what you ask for with `JsonnetOpts`.

Reply:
> // Peek loads the metadata of the environment at path. To get resources as well,
> // use Load

No, it explicitly mentions it only returns metadata